### PR TITLE
Fixes on dapp page

### PR DESCRIPTION
**Pull request by sirius651**
**Pull Request Summary**

Fixes on dapp page. ([Algem Fix](https://www.notion.so/astarnetwork/dApp-Staking-PP-V2-Algem-Fix-6484272369884b3b9af5fafdc00cc6da))

**Check list**
- [ ] contains breaking changes
- [ ] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

### CODE REVIEW

**Comment by bobo-k2** [comment]
image
Social icons for Algem are not aligned and there are duplicates

File: src/i18n/en-US/index.ts [hunk 1]
```diff
@@ -308,6 +312,7 @@ export default {
       },
       builder: {
         title: 'Builder',
+        githubAccount: 'Github account',
```
Comment (impelcrypto) [inline src/i18n/en-US/index.ts:315]:
```suggestion
        githubAccount: 'GitHub account',
```

File: src/components/dapp-staking/dapp/ProjectDetails.vue [hunk 2]
```diff
@@ -101,10 +101,22 @@
         </div>
         <div class="row--social-icons">
           <div v-for="(it, index) in communities" :key="index">
-            <button v-if="it.type === 'Twitter'" class="box--share btn--primary">
+            <button v-if="it.type === CommunityType.Discord" class="box--share btn--primary">
```
Comment (impelcrypto) [inline src/components/dapp-staking/dapp/ProjectDetails.vue:104]: This row should be aligned as one line
image

File: src/components/dapp-staking/dapp/ProjectDetails.vue [hunk 3]
```diff
@@ -113,52 +125,55 @@
...
+                  <astar-icon-base :icon-name="CommunityType.Reddit">
+                    <astar-icon-reddit />
+                  </astar-icon-base>
+                </a>
+              </div>
+              <q-tooltip>
+                <span class="text--tooltip">{{ $t('common.reddit') }}</span>
+              </q-tooltip>
+            </button>
+            <button class="box--share btn--primary">
```
Comment (impelcrypto) [inline src/components/dapp-staking/dapp/ProjectDetails.vue:140]: There are 2 facebook icons. Perhaps add `v-if="it.type === CommunityType.Facebook"`?
image

File: src/components/dapp-staking/dapp/ProjectDetails.vue [hunk 4]
```diff
@@ -113,52 +125,55 @@
...
+              <div class="icon--social">
+                <a :href="it.handle" target="_blank" rel="noopener noreferrer">
```
Comment (impelcrypto) [inline src/components/dapp-staking/dapp/ProjectDetails.vue:142]: It looks like the facebook icon navigate to twitter
image

Comment (impelcrypto) [reply src/components/dapp-staking/dapp/ProjectDetails.vue:142]: And this navigates to YouTube
image

File: src/i18n/en-US/index.ts [hunk 5]
```diff
@@ -69,6 +69,10 @@ export default {
     discord: 'Discord',
     github: 'GitHub',
     linkedIn: 'LinkedIn',
+    reddit: 'Reddit',
+    facebook: 'Facebook',
+    instagram: 'Instagram',
+    youtube: 'Youtube',
```
Comment (impelcrypto) [inline src/i18n/en-US/index.ts:75]:
```suggestion
    youtube: 'YouTube',
```

File: src/components/dapp-staking/dapp/ProjectDetails.vue [hunk 6]
```diff
@@ -113,52 +125,55 @@
...
+                  <astar-icon-base viewBox="0 0 18 18" :icon-name="CommunityType.Facebook">
+                    <astar-icon-facebook />
```
Comment (impelcrypto) [inline src/components/dapp-staking/dapp/ProjectDetails.vue:144]: Hover effect for the Facebook icon looks a bit odd 🤔 CC @Kahonnohak 
image

File: src/components/dapp-staking/dapp/ProjectDetails.vue [hunk 7]
```diff
@@ -113,52 +125,55 @@
...
+              </div>
+              <q-tooltip>
+                <span class="text--tooltip">{{ $t('common.facebook') }}</span>
+              </q-tooltip>
+            </button>
+            <button v-if="it.type === CommunityType.YouTube" class="box--share btn--primary">
+              <div class="icon--social">
+                <a :href="it.handle" target="_blank" rel="noopener noreferrer">
+                  <astar-icon-base :icon-name="CommunityType.YouTube">
+                    <astar-icon-youtube />
```
Comment (impelcrypto) [inline src/components/dapp-staking/dapp/ProjectDetails.vue:156]: Let's fix the hover styling for YouTube icon
image

**Review by impelcrypto — CHANGES_REQUESTED** [review]

**Comment by Kahonnohak** [comment]
Sorry I have not provided the social icons yet, the figma is not ready. It is the hardest week for my youngest that I just have to go to school so often.

File: src/components/dapp-staking/dapp/ProjectDetails.vue (hunk 3 above)
Comment (sirius651) [reply src/components/dapp-staking/dapp/ProjectDetails.vue]: yes my mistakes, here's [fix](https://github.com/AstarNetwork/astar-apps/pull/575/commits/057fe236bc863d486a278f3032791956c2da2773)

File: src/components/dapp-staking/dapp/ProjectDetails.vue (hunk 4 above)
Comment (sirius651) [reply src/components/dapp-staking/dapp/ProjectDetails.vue:142]: it's related with above issue

File: src/i18n/en-US/index.ts (hunk 5 above)
Comment (sirius651) [reply src/i18n/en-US/index.ts]: Fixed [here](https://github.com/AstarNetwork/astar-apps/pull/575/commits/2fcfa65a8fc6d90269794487567fcfba28f5b73b)

**Comment by Kahonnohak** [comment]
Hi Sorry being slow. Added FB and Insta. Could you use them please? and Youtube does look broken the layout too.

Screenshot 2022-11-12 at 10 46 29

File: src/components/dapp-staking/dapp/ProjectDetails.vue (hunk 7 above)
Comment (sirius651) [reply src/components/dapp-staking/dapp/ProjectDetails.vue:156]: 스크린샷 2022-11-13 오후 9 55 17

Fixed like this

**Review by Kahonnohak — APPROVED** [review]
LGTM

**Review by impelcrypto — APPROVED** [review]

**Pull request merged**